### PR TITLE
Rename `prepare` to `install`

### LINKED ISSUE #242: Refactor `pctl prepare` to `pctl install`
**What feature/behavior/change do you want?**
Makes more sense now that we have `pctl add` and is consistent with wego/flux

### PR DESCRIPTION
**Pull request by aclevername**
### Description

Rename `pctl prepare` to `pctl install`.

closes #242 

### Checklist
- [ ] Added tests that cover your change
- [ ] Added/modified documentation as required (such as the `README.md`)
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] Added a `kind` label to the PR (e.g. `kind/feature`)

### CODE REVIEW

File: pkg/cluster/install.go [hunk 1]
```diff
@@ -22,13 +22,13 @@ import (
 
 const (
 	kubectlCmd = "kubectl"
-	// profiles bundles ready to be installed files under `prepare`. The rest of the resources
+	// profiles bundles ready to be installed files under `install`. The rest of the resources
 	// are left for manual configuration.
-	prepareManifestFile = "prepare.yaml"
+	installManifestFile = "prepare.yaml"
```
Comment (nikimanoledaki) [inline pkg/cluster/install.go:27]: this could also be renamed to `install.yaml` but this might be too nit. wdyt about this?

Comment (aclevername) [reply pkg/cluster/install.go:27]: can't do this until we cut a `profiles` release with the installation file called `install.yaml`. Going to do this separately :smile:

**Comment by aclevername** [comment]
good catches @nikimanoledaki :tada:

**Review by nikimanoledaki — APPROVED** [review]
LGTM! :)

**Pull request merged**